docs: add desktops to per-session-mfa page

zmb3 opened the pull request:
Backports required:
- branch/v9
- branch/v8 (for the FIPS commit only)

zmb3 committed:
docs: add desktops to per-session-mfa page

webvictim commented on docs/pages/access-controls/guides/per-session-mfa.mdx:
```diff
@@ -199,3 +205,5 @@ Current limitations for this feature are:
   use Application access. We're working on integrating per-session
   MFA checks for these clients.
+- For Desktop Access, only WebAuthn devices are supported. Teleport does not
+  support U2F devices for Desktop Access MFA
```
```suggestion
  support U2F devices for Desktop Access MFA.
```

zmb3 committed:
Update docs for FIPS users

In order to configure WebAuthn on FIPS builds, you must set
local_auth to false and second_factor to optional.

Fixes #11080

ptgott reviewed (approved):
Approved with some small suggestions.
